.github/workflows/build.yml

```diff
@@ -46,9 +46,6 @@ jobs:
           - 8.10.5
           - 8.10.6
           - 8.10.7
-          # - 9.0.1
-          # - 9.0.2
-          # - 9.2.1
```

Comment:
This documents which versions have CI issues.  For that reason I would like to keep these.